### PR TITLE
Make PRs and Issues views have same commands

### LINKED ISSUE #1744: Make PRs and Issues explorers have same commands
the issues explorer doesn't have the same commands as the PR explorer.

image

- collapse all
- settings gear - could bring up the `"githubIssues.queries"` setting

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #1744

### CODE REVIEW

**Comment by alexr00** [comment]
@RMacfarlane, while I was doing this I noticed that the Sign in isn't working right now. Steps:
Sign out from User icon near gear
Sign in through "Sign in" in Pull Requests vew
See that the view isn't updated.

File: package.json [hunk 1]
```diff
@@ -955,6 +964,11 @@
           "when": "view =~ /prStatus/",
           "group": "navigation"
         },
+        {
+          "command": "pr.configurePRViewlet",
```
Comment (RMacfarlane) [inline package.json:968]: I think we could combine this with the entry above for configurePRViewlet by changing the when clause there

**Comment by RMacfarlane** [comment]
@alexr00 ah, I see the problem - This only happens when logging out and back in again. In the credential store reset method, I always just set the API to undefined, `this.initialize()` should be called after that

**Comment by RMacfarlane** [comment]
I'll push a commit here to fix it

**Pull request merged**